py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -36,6 +36,7 @@ def batch_norm(
     training: torch.Tensor,
     momentum: torch.Tensor,
     eps: List[float],
+    cudnn_enabled: bool,
```

Comment:
Since `None` is a valid input to the layer and batch norm weight and biases, could you add support for those inputs being optional/None and update the types in the function header here? Specifically, it could just be something like:

```python
if weight is None:
    weight = 1.0

if bias is None:
    bias = 0.0
```